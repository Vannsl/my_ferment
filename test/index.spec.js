import { shallowMount } from '@vue/test-utils'
import index from '@/pages/index'

const factory = () => {
  return shallowMount(index)
}

describe('index', () => {
  it('is a Vue instance', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
