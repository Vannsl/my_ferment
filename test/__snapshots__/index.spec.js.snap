// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renders properly 1`] = `"<div><hero-stub></hero-stub> <features-stub></features-stub> <teasers-stub></teasers-stub> <prices-stub></prices-stub> <call-to-action-stub></call-to-action-stub></div>"`;
